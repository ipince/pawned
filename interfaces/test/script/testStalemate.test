# A game in which a stalemate occurs

StartNewGame human 0 human 0
MakeMove f2-f4 0
MakeMove g7-g5 0
MakeMove f4-g5 0
MakeMove g8-f6 0
MakeMove g5-f6 0
MakeMove e7-f6 0
MakeMove e2-e3 0
MakeMove d7-d6 0
MakeMove d2-d4 0
MakeMove c7-c5 0
MakeMove d4-c5 0
MakeMove d6-c5 0
MakeMove d1-d8 0
MakeMove e8-d8 0
MakeMove c2-c4 0
MakeMove b7-b6 0
MakeMove b2-b3 0
MakeMove a7-a5 0
MakeMove a2-a4 0
MakeMove b6-b5 0
MakeMove c4-b5 0
MakeMove c8-h3 0
MakeMove g1-h3 0
MakeMove c5-c4 0
MakeMove b3-c4 0
MakeMove f6-f5 0
MakeMove e3-e4 0
MakeMove f5-e4 0
MakeMove c4-c5 0
MakeMove f8-c5 0
MakeMove c1-e3 0
MakeMove c5-e3 0
MakeMove b1-c3 0
MakeMove f7-f5 0
MakeMove c3-e4 0
MakeMove f5-e4 0
MakeMove h1-g1 0
MakeMove e3-g1 0
MakeMove h3-g1 0
MakeMove b8-c6 0
MakeMove b5-c6 0
MakeMove d8-c8 0
MakeMove c6-c7 0
MakeMove c8-c7 0
MakeMove g1-h3 0
MakeMove h8-g8 0
MakeMove e1-d1 0
MakeMove g8-g2 0
MakeMove f1-g2 0
MakeMove e4-e3 0
MakeMove g2-a8 0
MakeMove c7-d7 0
MakeMove d1-c1 0
MakeMove e3-e2 0
MakeMove c1-b1 0
MakeMove e2-e1 0
MakeMove b1-c2 0
MakeMove e1-a1 0
MakeMove a8-h1 0
MakeMove a1-h1 0
MakeMove h3-g1 0
MakeMove h1-g1 0
MakeMove c2-b2 0
MakeMove g1-h2 0
MakeMove b2-c1 0
MakeMove h2-b8 0
MakeMove c1-d1 0
MakeMove h7-h5 0
MakeMove d1-e1 0
MakeMove h5-h4 0
MakeMove e1-f1 0
MakeMove h4-h3 0
MakeMove f1-g1 0
MakeMove b8-d8 0
MakeMove g1-h1 0
MakeMove h3-h2 0
MakeMove h1-h2 0
MakeMove d8-c8 0
MakeMove h2-g1 0
MakeMove d7-e6 0
MakeMove g1-f1 0
MakeMove e6-f5 0
MakeMove f1-g1 0
MakeMove f5-g4 0
MakeMove g1-f1 0
MakeMove g4-h3 0
MakeMove f1-g1 0
MakeMove c8-e6 0
MakeMove g1-f1 0
MakeMove e6-e3 0
# Black in stalemate, so white can move infinitely many times
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0
MakeMove h3-h2 0
MakeMove h2-h3 0