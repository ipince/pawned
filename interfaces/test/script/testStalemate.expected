# A game in which a stalemate occurs

New game started
f2-f4
g7-g5
f4-g5
g8-f6
g5-f6
e7-f6
e2-e3
d7-d6
d2-d4
c7-c5
d4-c5
d6-c5
d1-d8
e8-d8
c2-c4
b7-b6
b2-b3
a7-a5
a2-a4
b6-b5
c4-b5
c8-h3
g1-h3
c5-c4
b3-c4
f6-f5
e3-e4
f5-e4
c4-c5
f8-c5
c1-e3
c5-e3
b1-c3
f7-f5
c3-e4
f5-e4
h1-g1
e3-g1
h3-g1
b8-c6
b5-c6
d8-c8
c6-c7
c8-c7
g1-h3
h8-g8
e1-d1
g8-g2
f1-g2
e4-e3
g2-a8
c7-d7
d1-c1
e3-e2
c1-b1
e2-e1
b1-c2
e1-a1
a8-h1
a1-h1
h3-g1
h1-g1
c2-b2
g1-h2
b2-c1
h2-b8
c1-d1
h7-h5
d1-e1
h5-h4
e1-f1
h4-h3
f1-g1
b8-d8
g1-h1
h3-h2
h1-h2
d8-c8
h2-g1
d7-e6
g1-f1
e6-f5
f1-g1
f5-g4
g1-f1
g4-h3
f1-g1
c8-e6
g1-f1
e6-e3
# Black in stalemate, so white can move infinitely many times
h3-h2
h2-h3
h3-h2
h2-h3
h3-h2
h2-h3
h3-h2
h2-h3
h3-h2
h2-h3
h3-h2
h2-h3
h3-h2
h2-h3
h3-h2
h2-h3
h3-h2
h2-h3
